### ISSUE TITLE
varA etc should be able to click into display

### ISSUE THREAD

**Issue by cpseager** [tarling/blockly-wedo#9]
and hence display will print their current value

**Comment by tarling** [comment]
Could you rephrase / explain this a bit more please

**Comment by cpseager** [comment]
It would be nice to be able to do this in the message area

Distance = 
51

![capturedistance](https://cloud.githubusercontent.com/assets/7423187/11067617/5140a4b8-87c5-11e5-81fd-14b2fbf6a906.PNG)

so you need to be able to snap the sensorA/sensorB etc variable into display

**Comment by cpseager** [comment]
or for variables like this:

![capturevar](https://cloud.githubusercontent.com/assets/7423187/11067661/9d7aab6c-87c5-11e5-9cd0-3911ff907f17.PNG)

**Comment by tarling** [comment]
@cpseager Please try now

**Comment by cpseager** [comment]
snaps in but sim hangs and doesn't display any value

![capturevara](https://cloud.githubusercontent.com/assets/7423187/11088308/3e2ebaba-8859-11e5-9429-57c6c28d0e71.PNG)

**Comment by cpseager** [comment]
I think we need to define vars with an initial value of 0 e.g.

var varA = 0;

to prevent them showing 'undefined' - may also help in other commands when used a a loop counter or similar
